lib/test/mock_loader.js

```diff
@@ -95,11 +97,17 @@ async function resolve(specifier, context, nextResolve) {
   if (isBuiltin(specifier)) {
     mockSpecifier = ensureNodeScheme(specifier);
   } else {
-    // TODO(cjihrig): This try...catch should be replaced by defaultResolve(),
-    // but there are some edge cases that caused the tests to fail on Windows.
+    const format = defaultGetFormatWithoutErrors(
+      pathToFileURL(context.parentURL ?? specifier)
```

Comment:
I'm not sure this is correct, we're getting `specifier` from the previous hooks in the chain, so it's not guaranteed to be a path. In any case, we should add a comment explaining why we're doing this

Reply:
I think this is wrong, too. I'll delete the specifier from the fallback value.